_remake/main.js

```diff
@@ -114,27 +123,75 @@ app.use(expressSession({
   }
 }));
 
-
 // flash message middleware
 app.use(flash());
 
+// if app is multi tenant
+// open MySQL connection
+// and use bodyParser (needed for post requests in lib/init-service-routes.js)
+if (RemakeStore.isMultiTenant()) {
```

Comment:
Can we conditionally require `mysql` in here, so it's not at the top?

Reply:
Yes